pkg/agent/config/node_config.go

```diff
@@ -44,6 +45,15 @@ const (
 	IPv6ExtraOverhead = 20
 )
 
+var (
+	// ServiceGWHairpinIPv4 is a virtual IP to perform SNAT for traffic of Windows ClusterIP Services from host.
+	// Such traffic needs to enter and exit antrea-gw0. Without OVS SNAT, the reply's destination IP
+	// is antrea-gw0 IP and it will not go through OVS pipeline anymore. It is used in Windows conntrackCommitTable
+	// table.
+	ServiceGWHairpinIPv4    = net.ParseIP("169.254.169.253")
```

Comment:
You could rebase on main now as the PR for Linux has been merged

Reply:
Working on it.

Reply:
It's duplicate with `VirtualServiceIPv4` above, the first comment of which has covered it.

Reply:
Removed.